### PR TITLE
links: add page

### PR DESCRIPTION
**Pull request by suyashbhawsar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by suyashbhawsar** [comment]
@kbdharun

File: pages/linux/links.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# links
+
+> Command-line web browser.
+> More information: <http://links.twibright.com/>.
+
+- Visit a website:
+
+`links {{example.com}}`
```
Comment (CleanMachine1) [inline pages/linux/links.md:8]:
```suggestion
`links {{https://example.com}}`
```

File: pages/linux/links.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Apply restrictions for anonymous account:
+
+`links -anonymous {{example.com}}`
```
Comment (CleanMachine1) [inline pages/linux/links.md:12]:
```suggestion
`links -anonymous {{https://example.com}}`
```

File: pages/linux/links.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Enable Cookies (1 to enable):
+
+`links -enable-cookies <0/1> {{example.com}}`
```
Comment (CleanMachine1) [inline pages/linux/links.md:16]:
```suggestion
`links -enable-cookies {{0|1}} {{example.com}}`
```

File: pages/linux/links.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`Up arrow key, Down arrow key`
+
+- Go forwards and backwards one page:
+
+`Left arrow key, Right arrow key`
+
+- Exit:
+
+`q then y`
```
Comment (CleanMachine1) [inline pages/linux/links.md:28]: I don't think we should include these.

Comment (suyashbhawsar) [reply pages/linux/links.md:28]: Okay

Comment (suyashbhawsar) [reply pages/linux/links.md:28]: But, it's included in the `lynx` page

**Comment by suyashbhawsar** [comment]
@kbdharun can you review this? Also, what are your suggestions with the navigations, should it be included in the page?

File: pages/linux/links.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
```
Comment (kbdharun) [inline pages/linux/links.md:28]:
```suggestion
`q then y keys`
```

**Review by kbdharun — APPROVED** [review]
> @kbdharun can you review this? 

Sure

>Also, what are your suggestions with the navigations, should they be included on the page?

As @CleanMachine1 suggested we don't generally include those but for special cases like this we would make an exception as these shortcuts would be essential for users to navigate through the application.

File: pages/linux/links.md (hunk 5 above)
Comment (suyashbhawsar) [reply pages/linux/links.md:28]: I think it'd be better to keep this page similar to the `lynx` page.

Comment (kbdharun) [reply pages/linux/links.md:28]: Sure, I too think the same. I will resolve the conversation.

**Review by kbdharun — APPROVED** [review]

**Comment by suyashbhawsar** [comment]
@CleanMachine1 do you have any more suggestions for this page?

**Review by blueskyson — APPROVED** [review]

**Pull request merged**